src/Uno.UI.Wasm/ts/Clipboard.ts

```diff
@@ -17,5 +49,28 @@
 
 			return "ok";
 		}
+
+		public static getText(requestId: string): void {
+			if (!Clipboard.dispatchGetContent) {
+				Clipboard.dispatchGetContent = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchGetContent");
+			}
+
+			var nav = navigator as NavigatorClipboard;
+			if (nav.clipboard) {
+				var promise = nav.clipboard.readText();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'promise' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)